test3.js

```diff
@@ -0,0 +1,191 @@
+    throw exc;
+  }
+};
+
+/**
+ * Failed redeliveries bulk delete in db
+ *
+ * @async
+ * @function DeleteRepository
+ * @params {object}
+ * @returns {object}
+ * @author dev-team
+ */
+
+const DeleteRepository = async params => {
+  try {
+    const { RepoModel, query, deliveryQ, updateHookDelivery } = params;
+    const res = await RepoModel.deleteOne(query);
+    if (res) {
+      if (deliveryQ && updateHookDelivery) updateHookDelivery(deliveryQ);
+    } else {
+      Logger.log('error', JSON.stringify(res));
+    }
+    return res;
+  } catch (exc) {
+    Logger.log('error', `Error in DeleteRepository in ${path.basename(__filename)}: ${JSON.stringify(exc)}`);
+    throw exc;
+  }
+};
+
+/**
+ * Failed redeliveries bulk delete in db
+ *
+ * @async
+ * @function AddOrModifiyRepoInDB
+ * @params {object}
+ * @returns {object} - array of objects
+ * @author dev-team
+ */
+
+const AddOrModifiyRepoInDB = async params => {
+  try {
+    const { action, listObj, deliveryQ, setOrgPaginateAndOnboard, schedulerEnabled, updateHookDelivery, getCommitsFromDB } = params;
+    const {
+      delivery: { org, orgId }
+    } = deliveryQ;
+
+    const RepoModel = await GetRepoModel(org, schedulerEnabled);
+
+    const { repo_name: name, repo_id: id } = listObj;
+    Logger.log('info', `add or update repository from hooks : ${name}`);
+    const query = { repo_id: id };
+
+    const updateParams = { RepoModel, deliveryQ, updateHookDelivery };
+    if (action === 'deleted') {
+      const res = await DeleteRepository({ ...updateParams, query });
+      return res;
+    }
+
+    if (action !== 'created') {
+      // get commits data for the repo and update created at based on commits
+      const [commit] = await getCommitsFromDB({ orgName: org, repoId: id, repoName: name });
+      if (commit && new Date(commit.date) < new Date(listObj.created_at)) listObj.created_at = commit.date;
+    }
+
+    const res = await UpdateRepository({ ...updateParams, query, update: listObj, options: { upsert: true, setDefaultsOnInsert: true } });
+    if (res && action === 'created') await setOrgPaginateAndOnboard({ orgId });
+    return res;
+  } catch (exc) {
+    Logger.log('error', `Error in AddOrModifiyRepoInDB in ${path.basename(__filename)}: ${JSON.stringify(exc)}`);
+    throw exc;
+  }
+};
+
+/**
+ * Update repository updated_at from hooks
+ *
+ * @async
+ * @function SetRepoUpdateAtInDB
+ * @params {object}
+ * @returns {object} - array of objects
+ * @author dev-team
+ */
+
+const SetRepoUpdateAtInDB = async params => {
+  try {
+    Logger.log('info', `Update repository updated_at from hooks`);
+    const { id, updateAt, org } = params;
+
+    const RepoModel = await GetRepoModel(org);
+
+    const query = { repo_id: id };
+
+    const repo = await RepoModel.findOne(query);
+    if (!repo) {
+      Logger.log('info', `Update repository updated_at from hook rebo ${id} not found`);
```

Comment:
Codacy has a fix for the issue: [Replace `'info',·`Update·repository·updated_at·from·hook·rebo·${id}·not·found`` with `⏎········"info",⏎········`Update·repository·updated_at·from·hook·rebo·${id}·not·found`⏎······`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650715)
```suggestion
      Logger.log(
```